libs/MVS/SceneRefine.cpp

```diff
@@ -424,6 +424,7 @@ void MeshRefine::ListCameraFaces()
 			Mesh::FacesInserter inserter(arrCameraFaces[ID]);
 			octree.Traverse(frustum, inserter);
 		}
+        octree.Release();
```

Comment:
none of these explicit releases are needed, the memory is released automatically by the destructor

Reply:
If you rely on destructors, it is recommended to make the Release method private

Reply:
no, Release() is still useful to call manually when you want to reuse the object, but it is also called automatically at destruction